assets/app/view/game/round/auction.rb

```diff
@@ -32,12 +33,14 @@ def render
 
           store(:before_process_pass, -> { hide! }, skip: true) if @current_actions.include?('pass')
 
-          if @current_actions.include?('par')
+          if @current_actions.include?('par') && @step.companies_pending_par
```

Comment:
this change looks too invasive i wonder if it needs another dedicated view or a different approach

Reply:
I was hesitant to add a new view just for a single game. It would also require a new round type, since that is what the game_page view uses to select the round view. None of that is hard, but I was trying to avoid game-specific view classes. I have tried to keep the changes to the shared parts auction view to a minimum.

I'll let you make the call as to whether a view used by just one game is the way to go.

Reply:
See below. I think I now agree we need a new view. The problem is that it is still an auction, but one that violates a lot of assumptions that the current auction view was written with (e.g. bids are always associated with a private company).

Reply:
Updated step to use auction view with only a few changes